package.json

```diff
@@ -39,11 +39,11 @@
     "test-legacy-javascript": "bash lighthouse-core/scripts/test-legacy-javascript.sh",
     "test-docs": "yarn --cwd docs/recipes/auth && jest docs/recipes/integration-test && yarn --cwd docs/recipes/custom-gatherer-puppeteer test",
     "test-proto": "yarn compile-proto && yarn build-proto-roundtrip",
-    "unit-core": "jest \"lighthouse-core/test/audits\"",
+    "unit-core": "jest \"lighthouse-core\"",
     "unit-cli": "jest \"lighthouse-cli/\"",
     "unit-viewer": "jest \"lighthouse-viewer/.*-test.js\"",
     "unit": "yarn unit-core && yarn unit-cli && yarn unit-viewer",
-    "unit:ci": "NODE_OPTIONS=--max-old-space-size=4096 npm run unit-core -- --ci && npm run unit-cli -- --ci && npm run unit-viewer -- --ci",
+    "unit:ci": "NODE_OPTIONS=--max-old-space-size=8192 npm run unit-core -- --ci && npm run unit-cli -- --ci && npm run unit-viewer -- --ci",
```

Comment:
I don't think that's going to be available on the github actions runners.

Seems like something bad is going on?

Maybe try out

```
--logHeapUsage
Logs the heap usage after every test. Useful to debug memory leaks. Use together with --runInBand and --expose-gc in node.
```

https://jestjs.io/docs/en/cli#--logheapusage

to see what it looks like?

Reply:
oops, probably should have removed `land-when-ci-is-green`. I assume this is going to continue to flake.

(and supposedly the [actions runners have 7GB RAM](https://docs.github.com/en/free-pro-team@latest/actions/reference/specifications-for-github-hosted-runners#supported-runners-and-hardware-resources), but I guess `--max-old-space-size` can handle that situation)